_data/talks.yml

```diff
@@ -1,4 +1,20 @@
 # Data file that describes talks
+
+- name: 'Web-APIs: Das ultimative Handbuch'
+  author: 'simon'
+  summary: >
+    Als Webentwickler verwendet man jeden Tag verschiedene Web-APIs. Häufig handelt es sich um die
+    allgemein bekannten APIs, wie das Document Object Model (DOM), das DOM Event API, das Fetch API
+    oder die Geolocation API. Es entsteht eine Art Komfortzone, die man selten verlässt. Dabei stellt
+    das Web noch viel mehr APIs zur Verfügung, die wenig oder gar nicht bekannt sind. Kennen Sie zum
+    Beispiel schon das Web Speech API, oder das Web Bluetooth API, das Server Timing API, das Battery
+    Status API oder das Credential Management API? Diese und weitere APIs sind Teil des Vortrags,
+    der Webentwicklern helfen soll, ihre Komfortzone zu verlassen. Da der Vortrag sehr viele APIs
+    umfasst, kann das Publikum am Anfang auswählen, welche APIs im Detail vorgestellt werden.
```

Comment:
```suggestion
Web developers use different Web APIs every day. Often they are the well known APIs, 
like the Document Object Model (DOM), the DOM Event API, the Fetch API or the 
Geolocation API. A kind of comfort zone is created, which one rarely leaves. 
The web provides many more APIs, which are rarely or not at all known. 
Do you already know for example the Web Speech API, or the Web Bluetooth 
API, the Server Timing API, the Battery Status API or the Credential Management 
API? These and other APIs are part of the talk to help web developers leave their 
comfort zone. Since the presentation includes many APIs, the attendees can choose 
at the beginning which APIs will be presented in detail.
```